common/djangoapps/student/management/tests/test_bulk_user_activate.py

```diff
@@ -0,0 +1,67 @@
+"""Tests for the bulk_user_activate command."""
+
+import ddt
+from django.core.management import call_command
+from django.core.management.base import CommandError
+
+from student.tests.factories import UserFactory
+from student.models import User
```

Comment:
This is an invalid import, should be `from django.contrib.auth.models import User`